README.md

```diff
@@ -225,8 +225,8 @@ end
 Route patterns may have optional parameters:
 
 ``` ruby
-get '/posts.?:format?' do
-  # matches "GET /posts" and any extension "GET /posts.json", "GET /posts.xml" etc.
+get '/post/:format?' do
+  # matches "GET /posts" and any extension "GET /posts/json", "GET /posts/xml" etc
```

Comment:
Additionally, the proposed change would not match `"GET /posts"`, but `"GET /post/"`.

How about something that would be more typically used as a path fragment? Perhaps

```ruby
get '/posts/?:year?' do
  # matches "GET /posts" and any year "GET /posts/2009", "GET /posts/2010", etc
end
```